calculator/src/Components/DisplayButtons.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+export default function DisplayButtons({num1,num2,toggleNum,handleNumber,handleNegative,handleClear,handleMath,handleEqual,handlePercent}) {
+    let displayNum = num1
+    if(num2 !== 0){
+        displayNum = num2
+    }else{
+        displayNum = num1
+    }
+    return (
+        <div className="calculator">
+            <p className="display">{Number(displayNum).toLocaleString('en-US')}</p>
```

Comment:
Display here could be it's own component. Not sure if it belongs in a component called DisplayButtons. Maybe Layout would be a more apt name?